rma_repair/README.rst

```diff
@@ -1,12 +1,39 @@
-.. image:: https://img.shields.io/badge/licence-LGPL--3-blue.svg
-    :alt: License LGPL-3
 
 ==========
 RMA Repair
 ==========
 
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
```

Comment:
Fragmented readme only applies to OCA repositories, sadly in Eficent readmes need to be manually done. In this case original readme I think is fine, no need to change because the menus and workflow stays unchanged.